### DISCUSSION TITLE
How to set Wireframe Mode for a Selection of Objects Only?

### DISCUSSION THREAD

**Discussion by GenieTim** [jimy-byerley/pymadcad#81] [Q&A] [answered]
This is related to Discussion#20, in that I am interested in making an object transparent. The Wireframe mode is perfectly fine for that. But, all the examples only show how to set everything to wireframe mode. 

Is there a way that I have not found yet how to set only certain meshes to wireframe mode?

**Comment by jimy-byerley** [comment, marked as answer]
There is no way to do it at the scene initialization for now ...
It is possible to do it at scene's runtime (when already displayed in a window), by changing the `options` dictionnary of the  instance of `madcad.displays.SolidDisplay` displaying the mesh. 

It shouldn't be too difficult to make it possible at initialization time :thinking:. like adding a `display_faces` attribute to `mesh.options` that is passed to `SolidDisplay` when loaded into a scene.
So we could write `mesh.option(color=vec3(..), display_faces=False)`

Generally speaking, more display options should be put in `mesh.options`, not only transparency
